helm/bundles/cortex-manila/prometheus-rules/manila.alerts

```diff
@@ -0,0 +1,183 @@
+  groups:
+  - name: cortex-manila-alerts
+    rules:
+    - alert: CortexManilaSchedulerDown
+      expr: |
+        up{component="cortex-manila-scheduler"} != 1 or
+        absent(up{component="cortex-manila-scheduler"})
+      for: 1m
+      labels:
+        context: liveness
```

Comment:
I started decoupling the alerts so they can be easily reviewed and linted (promtool). 
The content is unchanged.

Be aware that the `context` label is used for inhibition in combination with other labels.
Using different contexts, will nullify that.
Example: We want to inhibit elevated CPU utilization in favor of the service is down.

Reply:
same as [above](https://github.com/cobaltcore-dev/cortex/pull/215#discussion_r2225556389)